app/views/registration/steps/_details.html.haml

```diff
@@ -57,8 +57,25 @@
                 %select.chunky{ id: 'enterprise_country', name: 'country', required: true, ng: { init: "setDefaultCountry(#{Spree::Config[:default_country_id]})", model: 'enterprise.country', options: 'c as c.name for c in countries' } }
                 %span.error{ ng: { show: "details.country.$error.required && submitted" } }
                   = t(".country_field_error")
-
+      %div{ ng: {if: "enableMapConfirm"}} 
```

Comment:
This is great, but it still actually shows the map component on the page if the API key is not actually configured or present (eg the map is not working).

Reply:
I tried, but I can't figure out how to get the ENV var to work in this context. Do you know what I have to import to be able to check the key?